test/CMakeLists.txt

```diff
@@ -47,7 +47,15 @@ set(SRC
 )
 
 if (NGRAPH_ONNX_IMPORT_ENABLE)
-    list(APPEND SRC onnx_import.cpp)
+    if (APPLE OR WIN32)
+        list(APPEND SRC onnx_import.cpp)
+    else()
```

Comment:
Change else to if linux.

Reply:
There is no `if(LINUX)` in CMake. Only `if(UNIX)`, but is set for both Linux and Apple.